src/models/ItemTableData.ts

```diff
@@ -35,20 +36,24 @@ export default class ItemTableData {
       ...(this.showVolumeColumn() ? [this.volumeColumnHeading()] : []),
       "Format",
       ...(this.showAccessColumn() ? ["Access"] : []),
-      "Call Number",
-      "Item Location",
+      "Call number",
+      "Item location",
     ]
   }
 
-  get tableData(): (string | ReactElement)[][] {
+  get tableData(): ReactElement[][] {
     return this.items?.map((item) => {
       return [
         ...(this.showStatusColumn() ? [StatusLinks({ item })] : []),
-        ...(this.showVolumeColumn() ? [item.volume] : []),
-        item.format,
-        ...(this.showAccessColumn() ? [item.accessMessage] : []),
-        item.callNumber,
-        item.location.prefLabel,
+        ...(this.showVolumeColumn()
+          ? [ItemTableCell({ children: item.volume })]
+          : []),
+        ItemTableCell({ children: item.format }),
```

Comment:
This pops out because `ItemTableCell` is a component but the syntax here is more function-like. Not sure there's much you can do about this but just pointing it out.